package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "@js-temporal/polyfill": "^0.5.1"
+  }
+}
```

Comment:
We should have a package.json in each node package directory, but probably not at the top level